tests/unit/jvm_import/tags_propagator.bzl

```diff
@@ -0,0 +1,19 @@
+"""
+This module defines an aspect that propagates tags to help with testing jvm_import
+"""
```

Comment:
It would probably be similar to keep this in the `jvm_import_defs.bzl` file you've created above.